README.md

```diff
@@ -105,6 +107,7 @@ nmap <unique> <leader>pt <Plug>PickerTabedit
 nmap <unique> <leader>pv <Plug>PickerVsplit
 nmap <unique> <leader>pb <Plug>PickerBuffer
 nmap <unique> <leader>p] <Plug>PickerTag
+nmap <unique> <leader>pb] <Plug>PickerBufferTag
```

Comment:
Whilst testing this I found the longer mapping for `PickerBufferTag` a bit cumbersome. The `ctags` documentation uses the term "object" to refer to the symbols it indexes, so how about using `<leader>po`?